scheduler/plugin/guest.go

```diff
@@ -207,6 +213,30 @@ func (g *guest) normalizeScore(ctx context.Context) (framework.NodeScoreList, *f
 	return normalizedScoreList, framework.NewStatus(framework.Code(statusCode), statusReason)
 }
 
+// reserve calls guestExportReserve.
+func (g *guest) reserve(ctx context.Context) *framework.Status {
+	defer g.out.Reset()
+	callStack := g.callStack
+
+	if err := g.reserveFn.CallWithStack(ctx, callStack); err != nil {
+		return framework.AsStatus(decorateError(g.out, guestExportReserve, err))
+	}
+
+	statusCode := int32(callStack[0])
+	statusReason := paramsFromContext(ctx).resultStatusReason
+	return framework.NewStatus(framework.Code(statusCode), statusReason)
+}
+
+// unreserve calls guestExportUnreserve.
+func (g *guest) unreserve(ctx context.Context) {
+	defer g.out.Reset()
+	callStack := g.callStack
+
+	if err := g.unreserveFn.CallWithStack(ctx, callStack); err != nil {
+		klog.Error(decorateError(g.out, guestExportUnreserve, err))
```

Comment:
nit:
Please use contextual logging, not directly use klog.Error.
logger := klog.FromContext(ctx)
logger.Error()

Reply:
🤦‍♂
459a5a146c842e5c4730b5b1246bc84280499cca